pylinalg/func/vector.py

```diff
@@ -116,3 +116,114 @@ def vector_apply_matrix(vectors, matrix, /, *, w=1, out=None, dtype=None):
     if dtype is not None:
         out = out.astype(dtype, copy=False)
     return out
+
+
+def project_camera(vector, projection_matrix) -> np.ndarray:
+    """Project a 3D vector into 2D space.
```

Comment:
looking at the implementation in pygfx, this is really just applying a matrix to a vector. I think it doesn't need a function here.

Reply:
I agree. I would actually cull it completely and not introduce an alias.

Reply:
Me too!